state-chain/custom-rpc/src/lib.rs

```diff
@@ -369,15 +384,22 @@ pub trait CustomApi {
 	#[method(name = "pool_price")]
 	fn cf_pool_price(
 		&self,
-		from_asset: RpcAsset,
-		to_asset: RpcAsset,
+		from: RpcAsset,
```

Comment:
All the existing rpcs, need to be unchanged. The parameter names here are exposed as callers can provide json objects for the argument list i.e. { "from": "ETH"}. So changing the names of these will break the exising LP's code. Which we cannot do.

Whereas the new RPCs can have any naming, i.e. either "from" or "from_asset", because no one is using them.